### PR TITLE
New confirmation pane

### PR DESCRIPTION
**Pull request by alexvandesande**

### CODE REVIEW

**Comment by alexvandesande** [comment]
Ready to be merged

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
```
Comment (luclu) [inline .editorconfig]: The editor config can be only locally excluded by using `.git/info/exclude` instead of `.gitignore` (same syntax).

**Comment by luclu** [comment]
@alexvandesande here is some cropped text-field I ran into:

 bildschirmfoto 2016-09-08 um 22 44 10

**Comment by luclu** [comment]
There seems to be some issue with the SSL access to 4bytes when trying to decode the data (main-net and test-net):

bildschirmfoto 2016-09-08 um 23 34 59

Also at some point in time using this branch I could not input any text in any text-field anymore until I restarted Ethereum-Wallet. I will try to observe this and reproduce.

**Pull request merged**